examples/powercli/hostmaint-alarms/stack.yml

```diff
@@ -14,16 +14,4 @@ functions:
     secrets:
       - vc-hostmaint-config
     annotations:
-      topic: EnteredMaintenanceModeEvent
-  powercli-exitmaint:
-    lang: powercli
-    handler: ./handler
-    image: vmware/veba-powercli-esx-maintenance:latest
-    environment:
-      write_debug: true
-      read_debug: true
-      function_debug: false
-    secrets:
-      - vc-hostmaint-config
-    annotations:
-      topic: ExitMaintenanceModeEvent
\ No newline at end of file
+      topic: EnteredMaintenanceModeEvent, ExitMaintenanceModeEvent
```

Comment:
Can you please double-check whether this actually works, i.e. no `""` when multiple events are listed and space in between `", "`? `of-echo` should be fine to test with.

Reply:
Also, I think the README needs to be updated as well.